modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-adding-registry-pull-secret_{context}"]
+= Adding the registry to your pull secret
+
+Modify your the pull secret for your {product-title} cluster to describe
+your local registry before you install an {product-title} cluster in a
+disconnected environment.
+
+.Prerequisites
+
+* You prepared the bastion host to use in a disconnected installation.
+* You configured a mirror registry to use in your disconnected environment.
+
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Download your `registry.redhat.io` pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page.
+
+. Generate the base64-encoded user name and password or token for your mirror
+registry:
++
+----
+$ echo -n '<user_name>:<password>' | base64 -w0 <1>
+
+BGVtbYk3ZHAtqXs=
+----
+<1> For `<user_name>` and `<password>`, specify the user name and password that
+you configured for your registry.
+
+. Make a copy of your pull secret:
++
+----
+$ cat ./pull-secret.text | jq .  > <path>/pull-secret-2.json <1>
+----
+<1> Specify the path to the folder to store the pull secret in and a name for
+the file.
++
+The contents of the file resemble the following example:
++
+----
+{
+  "auths": {
+    "cloud.openshift.com": {
+      "auth": "b3BlbnNo...",
+      "email": "you@example.com"
+    },
+    "quay.io": {
+      "auth": "b3BlbnNo...",
+      "email": "you@example.com"
+    },
+    "registry.connect.redhat.com": {
+      "auth": "NTE3Njg5Nj...",
+      "email": "you@example.com"
+    },
+    "registry.redhat.io": {
+      "auth": "NTE3Njg5Nj...",
+      "email": "you@example.com"
+    }
+  }
+}
+----
+
+. Edit the new file and add a section that describes your registry to it:
```

Comment:
Do we care about the cloud.redhat.com pull-secret entries?  You'll need those to create the mirror, and to submit telemetry.  So maybe for setups where you had a proxy.  But for a truly disconnected install I don't see how the cluster would actually use them, which would allow simpler docs like:

> Create a pull secret containing only your local registry authentication information...

Reply:
Probably not. I'll make a separate module with the simpler workflow.